webapp/src/pages/home/Home.js

```diff
@@ -116,26 +124,31 @@ class Home extends Component {
                 <h4 className={this.state.applicationStatus == "Submitted" ? "text-success" : "text-danger"}>
                   {this.state.applicationStatus}
                 </h4>
-                
-                {this.state.applicationStatus == "Submitted" && <p>
-                  Thank you! Your application has been received and is under review. You can expect to hear our decision mid-June.
-                </p>}
+
+                {this.state.applicationStatus == "Submitted" &&
+                <div>
+                  <p>
+                    Thank you! Your application has been received and is under review. You can expect to hear our decision mid-June.
+                  </p>
+                  {this.state.offer !== null ? <p>There is an offer waiting for you, <NavLink to="/offer">click here</NavLink> to view it.</p>
+                  : <p>You are currently on the waiting list for the Deep Learning Indaba 2019. Please await further communication.</p>}
+                </div>}
                 {this.state.applicationStatus == "Withdrawn" && <p>
                   Your application has been withdrawn - you will not be considered for a place at the Indaba.
                 </p>}
                 {this.state.applicationStatus == "NOT Submitted" && <p>
                   You have NOT submitted your application! <NavLink to="/applicationForm">Complete and submit</NavLink> it before 26 May to be considered for a place at the Indaba.
```

Comment:
Please remove the links to applicationForm as these are no longer relevant :)

Reply:
what should be the remaining text there

> You have NOT submitted your application!
?